pkgs/development/python-modules/wxPython/4.0.nix

```diff
@@ -1,16 +1,31 @@
 { lib
+, stdenv
+, openglSupport ? true
+, libX11
+, pyopengl
 , buildPythonPackage
 , fetchPypi
 , pkgconfig
-, gtk3
 , libjpeg
 , libtiff
+, wxmac
+, darwin
 , SDL
 , gst-plugins-base
 , libnotify
 , freeglut
 , xorg
 , which
+, cairo
+, pango
+, pathlib2
+, python3
+, doxygen
+, ncurses
+, libpng
+, gstreamer
+, wxGTK
+, python3Packages
```

Comment:
don't import this directly, you inherit all the packages from they pkgs/development/python-modules/ directory with their respective python version

Reply:
thx! good to know